### PR TITLE
Create view for singe listing #84

### PR DESCRIPTION
**Pull request by OscarThorburn**

### CODE REVIEW

**Comment by OscarThorburn** [comment]
[US2002: Användare - klicka på annons](https://trello.com/c/MDod5i7x/39-us2002-anv%C3%A4ndare-klicka-p%C3%A5-annons)

File: src/GT.UI/Views/Listing/Listing.cshtml [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+﻿@model GT.Core.DTO.Impl.ListingDTO
 @{
+	ViewData["Title"] = "Annons";
+
 }
+
+<h1>Annons</h1>
+
+<h2>@Model.ListingTitle</h2>
+<div class="ListingData">
```
Comment (antoncarlgren) [inline src/GT.UI/Views/Listing/Listing.cshtml:10]: Shouldn't HTML class names generally be lower case?

**Review by antoncarlgren — CHANGES_REQUESTED** [review]

File: src/GT.UI/Views/Listing/Listing.cshtml (hunk 1 above)
Comment (OscarThorburn) [reply src/GT.UI/Views/Listing/Listing.cshtml:10]: Fixed

**Review by JohanSaisa — APPROVED** [review]
Approve

**Pull request merged**